features/update/set_price.feature

```diff
@@ -16,10 +16,10 @@ Feature: Update price fields
       | BOX4 |
       | BOX5 |
     Then I should get the following products after apply the following updater to it:
-      | product | actions                                                                                                                                                                                                                                          | result                                                                                                                                       |
+      | product | actions                                                                                                                                                                                                        | result                                                                  |
       | BOX1    | [{"type": "set_data", "field": "price", "data": [{"amount": "12.4", "currency": "EUR"}], "locale": "fr_FR", "scope": null}]                                                                                                                      | {"values": {"price": [{"locale": "fr_FR", "scope": null, "data": [{"amount": 12.4, "currency": "EUR"}]}]}}                                   |
       | BOX2    | [{"type": "set_data", "field": "price", "data": [{"amount": 5, "currency": "EUR"}], "locale": "fr_FR", "scope": null}]                                                                                                                           | {"values": {"price": [{"locale": "fr_FR", "scope": null, "data": [{"amount": 5, "currency": "EUR"}]}]}}                                      |
       | BOX3    | [{"type": "set_data", "field": "price", "data": [{"amount": 5.3, "currency": "EUR"}], "locale": "fr_FR", "scope": null}]                                                                                                                         | {"values": {"price": [{"locale": "fr_FR", "scope": null, "data": [{"amount": 5.3, "currency": "EUR"}]}]}}                                    |
       | BOX4    | [{"type": "set_data", "field": "price", "data": [{"amount": "5", "currency": "USD"}], "locale": "fr_FR", "scope": null}]                                                                                                                         | {"values": {"price": [{"locale": "fr_FR", "scope": null, "data": [{"amount": 5, "currency": "USD"}]}]}}                                      |
-      | BOX4    | [{"type": "set_data", "field": "price", "data": [{"amount": "5", "currency": "EUR"}], "locale": "fr_FR", "scope": null}, {"type": "set_data", "field": "price", "data": [{"amount": "5", "currency": "USD"}], "locale": "fr_FR", "scope": null}] | {"values": {"price": [{"locale": "fr_FR", "scope": null, "data": [{"amount": null, "currency": "EUR"}, {"amount": 5, "currency": "USD"}]}]}} |
```

Comment:
Here, the new price collection generated from the updated does not keep the prices set to null. and generates a clean price collection with actual values in it.